Parsers/ASimAuditEvent/Parsers/ASimAuditEventVectraCloudAudit.yaml

```diff
@@ -0,0 +1,52 @@
+Parser:
+  Title: Audit Event ASIM parser for VectraCloud Audit Logs Event
+  Version: '0.1'
+  LastUpdated: Mar 17 2023
+Product:
+  Name: Vectra
+Normalization:
+  Schema: AuditEvent
```

Comment:
@devikamehra Vectra Audit data has additional data along with login/logout messages. We have included additional sample data for the Vectra Audit. This should give you a better understanding of the Vectra Audit data.

Reply:
@dhruvilbhatt-crest I still see only login events in the sample data - https://github.com/Azure/Azure-Sentinel/blob/aa8c2cceb4865cefd007be9a54775e0f5a6a7f82/Sample%20Data/Custom/VectraAudits.csv

Can you please guide me where the updated logs are?

Also, in case we have more than one type of data in the Audit logs, we can filter the data it in the parser and develop Authentication ASim Parser for authentication while AuditEvents ASim Parser for the rest.

Reply:
Hi @devikamehra, 

You can find the ASim sample data here: https://github.com/dhruvilbhatt-crest/Azure-Sentinel/tree/vectra-cloud/Sample%20Data/ASIM/VectraCloud

Per our understanding, the data will belong to Audit Events only. Upon reviewing the sample data available at the link above, we may discuss further.

Reply:
@dhruvilbhatt-crest Thank you for providing additional information. Wanted to confirm if we only get login success  events in this table or there can be logout or failures as well? If yes, in my opinion, it makes total sense to have an authentication parser for this table filtered on event_type.

For all the remaining event_types, Audit Schema is more appropriate and we should leverage object and operation fields for each event_type effectively.

Reply:
Hi @devikamehra, we have introduced Authentication Parser and made the necessary changes.